fix: corrigindo erro de exportação

katsudon02 opened the pull request:
Correção do problema de erro de exportação quando semestres planejados ou completos são vazios;
Agora o export usa dados reais.

katsudon02 committed:
fix: corrigindo erro de exportação

Teruya11 reviewed (dismissed):
Pronto para merge

KakeyaK commented on resources/views/exportTemplate.blade.php:
```diff
@@ -136,60 +136,90 @@
+                            <td>{{ $subject['name'] }}</td>
+                            <td class="num">{{ $subject['lecture_credits'] }}</td>
+                            <td class="num">{{ $subject['work_credits'] }}</td>
+                            <!-- <td class="num">100</td>
```
Apagar comentários

KakeyaK commented on resources/views/exportTemplate.blade.php:
```diff
@@ -136,60 +136,90 @@
+                        </tr>
+                        @endif
+                    @endforeach
+                    <!-- Totals row removed as per user request -->
```
Esse também

KakeyaK commented on resources/views/exportTemplate.blade.php:
```diff
@@ -136,60 +136,90 @@
+                        <th>Nome da Disciplina</th>
+                        <th>AU</th>
+                        <th>TR</th>
+                        <!-- <th>FREQ</th>
```
Esse também

KakeyaK commented on resources/views/exportTemplate.blade.php:
```diff
@@ -136,60 +136,90 @@
+                            <td>{{ $subject['name'] }}</td>
+                            <td class="num">{{ $subject['lecture_credits'] }}</td>
+                            <td class="num">{{ $subject['work_credits'] }}</td>
+                            <!-- <td class="num">100</td>
```
Esse

KakeyaK commented on resources/views/exportTemplate.blade.php:
```diff
@@ -136,60 +136,90 @@
+                        </tr>
+                        @endif
+                    @endforeach
+                    <!-- Totals row removed as per user request -->
```
Esse tb

KakeyaK reviewed (commented):
Melhor limpar os comentários antes de mergear

Teruya11 committed:
Cleaning unnecessary comments

Removed commented-out columns and totals row from the export template as per user request.

Signed-off-by: Daiki <133552179+Teruya11@users.noreply.github.com>

Teruya11 reviewed (approved):
Limpo
